src/main/java/Main.java

```diff
@@ -0,0 +1,31 @@
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private Luna luna = new Luna("./data/luna.txt");
+
+    @Override
+    public void start(Stage stage) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(Main.class.getResource("/view/MainWindow.fxml"));
+            AnchorPane ap = fxmlLoader.load();
+            Scene scene = new Scene(ap);
+            stage.setScene(scene);
+            stage.setTitle(" ❈  ✶  ✧︎ ✱✬ ✨  l u n a  ✨ ❇︎ ✫❍   ❈ ✶  ✶ ");
+            stage.setResizable(false);
+            fxmlLoader.<MainWindow>getController().setLuna(luna);
+            stage.show();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Perhaps the exception can be handled in a different way instead of printing the stack trace?